RELEASE_NOTE.txt

```diff
@@ -1,13 +1,53 @@
 Release Notes:
 ===============
 
-Google SAML Bridge for Windows
+SAML Bridge for Google Search Appliance
 
-This document contains the release notes for Google SAML Bridge for Windows.
+This document contains the release notes for SAML Bridge for Google Search Appliance.

```

Comment:
Here let's say "... for SAML Bridge for Windows, a part of the Google Search Appliance."